test/js/ui/map.test.js

```diff
@@ -930,6 +931,21 @@ test('Map', function(t) {
         t.end();
     });
 
+    t.test('workerCount option', function(t) {
+        var map = createMap({});
+        // TODO: it's not great to check this private member here; better would
+        // be to mock Style and then just check that the correct workerCount
+        // param is being passed.
+        t.equal(map._workerCount, browser.hardwareConcurrency - 1, 'workerCount defaults to hardwareConcurrency - 1');
+        map = createMap({ workerCount: 3 });
+        t.equal(map._workerCount, 3, 'workerCount option is used');
```

Comment:
How 'bout checking `map.style.dispatcher.actors.length` here? It also accesses private properties but is 🆗 for now.